File: LeetCode/Python/0026. Remove Duplicates from Sorted Array #1.py
class Solution:
    def removeDuplicates(self, arr: List[int]) -> int:
        n = len(arr)
        for i in range(1, n):
            if arr[i - 1] == arr[i]:
                arr[i - 1] = float("inf")
        
        arr.sort()
        return n - arr.count(float("inf"))

File: LeetCode/Java/0026. Remove Duplicates from Sorted Array #1.java
class Solution {
    public int removeDuplicates(int[] arr) {
        int n = arr.length;
        int count = 0;
        
        for (int i = 1; i < n; i++) 
            if (arr[i - 1] == arr[i]) {
                arr[i - 1] = Integer.MAX_VALUE;
                count++;
            }
        
        Arrays.sort(arr);
        return n - count;
    }
}
